src/applications/letters/components/NoAddressBanner.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const NoAddressBanner = (
+  <div className="usa-alert usa-alert-warning">
+    <div className="usa-alert-body">
+      <h3 className="usa-alert-heading">
+        VA does not have a valid address on file for you
+      </h3>
+      <div className="usa-alert-text">
+        You will need to update your address before we can provide you any
+        letters.
+      </div>
+    </div>
+  </div>
+);
+
+export default NoAddressBanner;
```

Comment:
Nit: Probably should be `noAddressBanner` because you couldn't do `<NoAddressBanner/>` with this. :man_shrugging:

(I knew the terminology to describe this once upon a time, but it escapes me right now. :sweat_smile:)